packages/trpc/server/routers/viewer/deploymentSetup.tsx

```diff
@@ -0,0 +1,27 @@
+import { DeploymentLicenseType } from "@prisma/client";
+import { z } from "zod";
+
+import prisma from "@calcom/prisma";
+
+import { router, authedProcedure } from "../../trpc";
+
+export const deploymentSetupRouter = router({
+  update: authedProcedure
+    .input(
+      z.object({
+        licenseType: z.nativeEnum(DeploymentLicenseType).optional(),
```

Comment:
What is `licenseType` being used for?

Reply:
to store the license type EE or FREE, if we choose to use `licenseKey` only, we can't distinguish between an admin that already went through the wizard or not

Reply:
How so? The wizard should display for empty license key

Reply:
Ow, that's right, we don't need to remember which license was selected because we want to allow the admin to go back in the wizard to change it, making sense of the "Previous Step" button. Yeah, we can remove the license type I guess.